app/views/race_editions/enter.html.erb

```diff
@@ -6,7 +6,7 @@
 
     <%= race_edition_form.fields_for :racers, @racer do |racer_fields| %>
       <%= racer_fields.label :first_name %>
-      <%= racer_fields.text_field :first_name %>
+      <%= racer_fields.text_field :first_name, autofocus: true %>
```

Comment:
Not strictly related to this PR, but I think this change marginally improves the user experience.

Reply:
Love it.